README.md

```diff
@@ -51,43 +38,138 @@ $ bundle install
 
-If you discover a potential security issue in this project we ask that you notify AWS/Amazon Security via our [vulnerability reporting page](https://aws.amazon.com/security/vulnerability-reporting/). Please do **not** create a public GitHub issue.
+The documentation should be informal, but not chatty.  Our goal is to produce concise, technically accurate content that helps the user accomplish a task.  It should not include future plans, what we were thinking during feature planning, or our reasoning behind implementation decisions. Such information should be in the product roadmap.
 
-## License
+Write in active voice, with the assumption that the reader is the person that will perform actions: “You should specify the password associated with...” or instructional:  “Specify the password associated with...”.  Focus on the user’s task, and the steps required to accomplish it.  If all of the steps or information is not included on one page, provide supporting links.
+
+When referring to proprietary products, use the complete company name and the product name once at the start of the page; thereafter, use only the product name. 
+
+  * The term Microsoft SQL Server is the complete company name and product name.  For example: Babelfish extends your PostgreSQL database with the ability to accept database connections from Microsoft SQL Server clients.
+  * The term SQL Server refers to the database, the database content, and transactions/procedures/functions that involve more than SQL syntax.  For example: Port 1433 accepts SQL Server commands, with the limitations discussed later in this guide.
+  * Use T-SQL when referring to syntax or language.  For example: You should use correct T-SQL syntax when forming commands.
+  * Use TDS when referring to the protocol used by SQL Server.  For example: The server listens on port 1433 for client connections in TDS protocol.
+
+To make the documentation easier to read and understand:
+
+    * Use short but complete sentences.  
+    * Be consistent with punctuation.
+    * Use bulleted lists and tables where appropriate.
+
+Avoid using:
+    * Jargon or Slang - whether marketing, developer, or internal terms
+    * Colloquialisms 
+    * Questions
+    * Latin (vs., e.g., i.e.,  etc)
+
+The Babelfish project supports an environment of diversity and equity.  In the interest of promoting inclusivity, the following words (unless used in a literal/strictly technical manner such as an object name or command) should be avoided.  Alternatives are recommended below.  Instead of: 
+    * execute → use start or run 
+    * abort → use stop
+    * enable → use turn on, activate, start
+    * disable → use turn off, deactivate, stop
+    * kill → use end or stop
+    * blacklist → use deny list
+    * master → use primary, main, or leader
+    * hang → use stop responding
+    * slave → use secondary, replica, or standby
+    * black day → use blocked day
```

Comment:
Needs to be removed; blocked days only make sense internally.

Reply:
Update committed: 904dbb0..d57fd49  readme_reworked -> readme_reworked